controller.js

```diff
@@ -46,3 +49,19 @@ exports.patchReview = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getReviews = (req, res, next) => {
+  const { category: filter } = req.query;
+  if (filter) {
+    checkIfCategoryExists(filter).catch((err) => {
```

Comment:
this code is all good =, but it doesnt look like you are testing for a 404 if the category query doesnt exsist or if we get back an empty array if the category exists but doesnt have any articles 🤔